pyquest_cffi/cheat/test_measurement.py

```diff
@@ -120,7 +120,7 @@ def test_get_simple() -> None:
     assert density_matrix == 0
     assert num_amps == 4
     assert num_qubits == 2
-    assert expec_val == 1
+    assert expec_val == 0.5
```

Comment:
I like that the new unit test has a correct result that is not 1, 1 would have a higher chance to appear by accident when the method implementations fails.